### PR TITLE
Beautify code again

### PR DESCRIPTION
**Pull request by skorokithakis**
This is a continuation of #696, which implements #693. After review says this is okay, I will beautify the code using the configs.

Also, I want to add this to CI, but I noticed the Travis file was removed. What do we use nowadays?

### CODE REVIEW

**Comment by yagebu** [comment]
> What do we use nowadays?

We use Github Actions now - configuration is in `.github/workflows/ci.yml`

File: .pre-commit-config.yaml
Comment (yagebu) [inline .pre-commit-config.yaml]: Is this required? flake8 supports setup.cfg as one of its default config files.

File: setup.cfg
Comment (yagebu) [inline setup.cfg]: Is this necessary? Shouldn't `pre-commit` only run on files that are checked into git?

File: .pre-commit-config.yaml
Comment (skorokithakis) [reply .pre-commit-config.yaml]: Hmm true, I'll remove it, thanks.

File: setup.cfg
Comment (skorokithakis) [reply setup.cfg]: It shouldn't be, I just add it to all my repos. Will remove.

**Comment by skorokithakis** [comment]
Is this ready to merge? I will run the actual linting after it's approved and right before we merge.

**Review by yagebu — APPROVED** [review]

**Comment by skorokithakis** [comment]
Hmm, does anyone know why lint failed? I didn't add any exec statements.

**Comment by runfalk** [comment]
Maybe the disable doesn't work when it's split across multiple lines?

https://github.com/lektor/lektor/pull/823/files#diff-75a248b337f2de305ae21d3819159e1e53ab6ca222c427a177cc759e128723a9R36-R38

**Comment by skorokithakis** [comment]
Ah, that'd do it, thanks @runfalk! Will fix.

File: lektor/admin/modules/api.py
Comment (runfalk) [inline lektor/admin/modules/api.py:5]: Which tool is causing these import splits? I don't like this idea.

Comment (skorokithakis) [reply lektor/admin/modules/api.py:5]: That's `reorder_python_imports`. What don't you like?

Comment (skorokithakis) [reply lektor/admin/modules/api.py:5]: It's done so diffs will be much simpler, you can see an added/removed line rather than hunt for a word in a long list.

Comment (runfalk) [reply lektor/admin/modules/api.py:5]: I don't like the noise it produces:

```python
# reorder_python_imports
from flask import abort
from flask import Blueprint
from flask import g
from flask import redirect
from flask import render_template
from flask import request
from flask import url_for

# black (don't know whether or not it sorts the functions alphabetically)
from flask import (
    Blueprint,
    render_template,
    abort,
    redirect, request,
    g,
    url_for,
)
```

I think the vanilla black style is much better (probably because it's similar to what I'm used to and that it's less verbose). I guess it also has the benefit of producing a smaller diff if we ever rename a file.

Comment (skorokithakis) [reply lektor/admin/modules/api.py:5]: Hmm okay, I'll revert the two commits, remove `reorder_python_imports` and redo them, though then we won't get the automatic separation of third party packages, consistent ordering, etc.

Comment (nixjdm) [reply lektor/admin/modules/api.py:5]: In other (large, unrelated) repos I've worked with, reorder-python-imports also has the effect of automatically fixing some flake8 errors, such as deduping imports, and ensuring the correct import order, which can introduce bugs, even if that's rare. Admittedly, that's less of a problem here since Lektor is small in comparison and in better shape.

I also didn't like splitting imports like this (or black, for that matter), but they've both grown on me a lot. My vote would be to keep reorder-python-imports. This single diff is a little noisy (thus the .git-blame-ignore-revs), but I think it's worth it. I now don't blink at the import lines that look like this. My 2 cents.

Comment (nixjdm) [reply lektor/admin/modules/api.py:5]: I think I typed my comment as you were typing your last one @skorokithakis . For the record, I'm ok with or without reorder-python-imports, so we can remove it if you and @runfalk want. No need to trip up this PR for that. Though I'd prefer we keep it.

Comment (skorokithakis) [reply lektor/admin/modules/api.py:5]: I would prefer to keep it too, given that I like it (which is why I added it) and how much work it would be to remove it (I'd have to redo half the PR).

My vote is to keep. This PR is ready if we do, though I don't understand why tests are failing on OS X, I think it's unrelated.

Comment (runfalk) [reply lektor/admin/modules/api.py:5]: macOS tests are flaky during to the environment setup (before anything lektor related is done). You can retrigger the tests if you want to be sure.

Though I'm still not sold on the import split I'm not gonna hold up the PR if no one else dislikes it. Maybe it's an aquired taste :laughing:

Comment (skorokithakis) [reply lektor/admin/modules/api.py:5]: I can certainly say I've acquired it and now prefer the determinism, @runfalk.

**Comment by skorokithakis** [comment]
Oof, these tests just don't want to pass.

**Comment by skorokithakis** [comment]
@runfalk / @nixjdm, tests won't pass, what should I do?

**Comment by runfalk** [comment]
You can retrigger them by going to the run's page (https://github.com/lektor/lektor/runs/1298211911). I've retriggered them just now. If they still don't work just merge anyway I guess. It would be very weird if this introduced a platform specific bug, and if it did I guess we just have to fix it later.

**Comment by skorokithakis** [comment]
@runfalk I've rerun them about five times :/ I think we should merge anyway, yeah...

**Comment by skorokithakis** [comment]
This error doesn't look like it's caused by this PR, or that it's going to go away soon:

```
Error: Fetching /usr/local/Homebrew/Library/Taps/local/homebrew-openssl failed!
Fetching /usr/local/Homebrew/Library/Taps/local/homebrew-python2 failed!
fatal: invalid upstream 'origin/master'
fatal: invalid upstream 'origin/master'
```

I wonder what happened with Homebrew...

**Comment by skorokithakis** [comment]
Merged, thank you all!

**Comment by nixjdm** [comment]
Yeah, I was just about to say it looks like this breakage is totally unrelated to the code changes. It broke when I reran the previous tip.

**Pull request merged**
